src/components/OSCALProfileCatalogInheritance.js

```diff
@@ -1,86 +1,49 @@
-import React, { useState } from "react";
-import { styled } from "@mui/material/styles";
-import { TreeItem, TreeView } from "@mui/lab";
+import React from "react";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
 import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
-import ExpandLessIcon from "@mui/icons-material/ExpandLess";
-import { Grid, IconButton, List, Paper, ListSubheader } from "@mui/material";
-
-const CatalogInheritancePaper = styled(Paper)(
-  ({ theme }) => `
-  margin-top: ${theme.spacing(2)};
-  margin-bottom: ${theme.spacing(2)};
-  position: relative;
-  overflow: auto;
-`
-);
-
-function generateLabel(title, type) {
-  return type === "profile" ? `Profile: ${title}` : `Catalog: ${title}`;
-}
-
-function createTree(tree, ids) {
-  if (tree.inherited == null) {
-    return null;
-  }
-
-  const children = [];
+import { TreeItem, TreeView } from "@mui/lab";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
 
-  tree.inherited.forEach((childProfileOrCatalog) => {
-    ids.push(`${childProfileOrCatalog.uuid} #${ids.length + 1}`);
-    children.push(
+export default function OSCALProfileCatalogInheritance(props) {
+  const inheritedListItem = (item) => {
+    if (item.type === "catalog") {
+      return (
+        <TreeItem
+          key={item.uuid}
+          nodeId={item.uuid}
+          label={`Catalog: ${item.title}`}
+        />
+      );
+    }
+    return (
       <TreeItem
-        nodeId={`${childProfileOrCatalog.uuid} #${ids.length}`}
-        key={`${childProfileOrCatalog.uuid} #${ids.length}`}
-        label={generateLabel(
-          childProfileOrCatalog.title,
-          childProfileOrCatalog.type
-        )}
+        key={item.uuid}
+        nodeId={item.uuid}
+        label={`Profile: ${item.title}`}
       >
-        {createTree(childProfileOrCatalog, ids)}
+        {item.inherited?.map((inherited) => inheritedListItem(inherited))}
       </TreeItem>
     );
-  });
-
-  return children;
-}
-
-export default function OSCALProfileCatalogInheritance(props) {
-  const [expandedIds, setExpandedIds] = useState([]);
-  const ids = [];
-
-  const children = createTree(props.inheritedProfilesAndCatalogs, ids);
-
-  if (ids.length > 0 && expandedIds.length === 0) {
-    setExpandedIds(ids);
-  }
+  };
```

Comment:
Okay now I am also kinda curious whether we can do something like:

```
const documentLabel = (
  <Typography>
  {item.title}
  <Chip label={item.type[0].toUpperCase() + item.type.slice(1) />
  </Typography>
);

return (
  <TreeItem
    key={item.uuid}
    nodeId={item.uuid}
    label={documentLabel}
  >
  {item.inherited?.map((inherited) => inheritedListItem(inherited))}
  </TreeItem>
);
```

Since the `children` would either be a list or `undefined` in this case, we don't really need the if statement for that. And the profile vs catalog thing can be solved by just semi-title-casing the document `type`. And moving it to `Chip` at the end of the text at the same time kinda cleans up the display a bit. And I think this reduced some duplication.